src/ignitus-Shared/Components/ignitus-DashBoard-Footer/style.scss

```diff
@@ -0,0 +1,91 @@
+@import '../../styles/ignitus-components-common.scss';
+@import '../../styles/ignitus-tokens.scss';
+
+.rectangle {
+  position: absolute;
+  width: 276px;
+  height: 121px;
+  left: 1072px;
```

Comment:
It would be great if we can rely on flex-box architecture instead of relying on `left`, `top`, and `width` properties like in #about, #privacy, #careers as well.

e.g -

What I meant is attached in the pic. if you have knowledge of flex-box then it will be very easy to design such kind of layouts and the best part is that it will be responsive.
`Flex-box` is my opinion you can also go with `css grids`.

![75677565-0eb2d800-5c8c-11ea-970d-6f373b03d65c](https://user-images.githubusercontent.com/13464678/76887888-3bb5dc00-6883-11ea-8303-c93ac30d405b.png)

Reply:
Got it. I'll make the changes and resubmit

Reply:
sure thanks 👍

Reply:
Yes, I think using flexbox would be good. 👍